Essentials/src/main/java/com/earth2me/essentials/commands/Commandpay.java

```diff
@@ -33,38 +38,37 @@ public void run(final Server server, final User user, final String commandLabel,
             throw new NotEnoughArgumentsException();
         }
 
-        final String ogStr = args[1];
```

Comment:
`ogStr` is a terrible name but it might still be nicer to keep a variable for clarity. Maybe rename to `amountStr`?